pulpanalytics/views.py

```diff
@@ -183,6 +183,17 @@ def _add_content_apps_data(context, daily_summary):
     )
 
 
+def _add_postgresql_version(context, daily_summary):
+    if daily_summary is None:
+        return
+
+    for postgresql_version_proto_field in daily_summary.summary.postgresql_version:
+        if postgresql_version_proto_field.version == "":
+            postgresql_version_proto_field.version = "None"
+        context["postgresql_versions_labels"].append(postgresql_version_proto_field.version)
+        context["postgresql_versions_count"].append(postgresql_version_proto_field.count)
```

Comment:
I thought we were going to bucket the results into known versions here and collect the rest as "unknown".

Reply:
Do you mean s/None/Unknown/. That would be a fine change to me. The unknown values in the DB are stored as `""` so what this is doing is making the label the user sees on the chart read as `None`. Making that `Unknown` is fine with me, just let me know what you think is best.

Reply:
What i mean is that the client can send us everything. But we only want to show versions of postgres, we know of. And all the others we sum up in "unknown". So what i expect to see here is the list of valid known versions of postgres like "15.4", "13.5", ...

Reply:
What i mean is that this app needs to decide when a valid version is provided. We cannot rely on the client sending proper data.